File: C#/Example Statement/Example If, Else If, dan Else/If, Else If, dan Else.cs
using System;

namespace Main
{
    class main
    {
        static void Main(string[] args)
        {
            int varchc;

            Console.WriteLine("Menu :\n");
            Console.WriteLine("Enter Number : 1 for Example If");
            Console.WriteLine("Enter Number : 2 for Example Else If");
            Console.WriteLine("Enter Number other 1 and 2 for Example Else\n");
            Console.Write("Please Enter Number : ");
            varchc = int.Parse(Console.ReadLine());

            if (varchc == 1)
            {
                Console.WriteLine("\nExample If");
            }
            else if (varchc == 2)
            {
                Console.WriteLine("\nExample Else If");
            }
            else
            {
                Console.WriteLine("\nExample Else");
            }

        }
    }
}

File: C++/Example Statement/Example If, Else If, dan Else/If, Else If, dan Else.cpp
#include <iostream>

using namespace std;

int main()
{
	int pil;
	cout << "Menu :\n" << endl;
	cout << "Enter Number : 1 for Example If" << endl;
	cout << "Enter Number : 2 for Example Else If" << endl;
	cout << "Enter Number other 1 and 2 for Example Else\n" << endl;
	cout << "Please Enter Number : ";
	cin >> pil;

	if (pil == 1) {
		cout << "\nExample If" << endl;
	}
	else if (pil == 2) {
		cout << "\nExample Else If" << endl;
	}
	else {
		cout << "\nExample Else" << endl;
	}

	return 0;
}
